src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const LONG_LEASE = 7;
+  const LONG_LEASE_DISCONT = 50;
+  const AVERAGE_LEASE = 3;
+  const AVERAGE_LEASE_DISCONT = 20;
+  const PRICCE_FOR_DAY = 40;
```

Comment:
There is a typo in the constant name `PRICCE_FOR_DAY`. It should be `PRICE_FOR_DAY` to correctly reflect its purpose.